### DISCUSSION TITLE
I have a question about sequential execution for multiple flows

### DISCUSSION THREAD

**Discussion by lHuKKal** [postmanlabs/postman-flows#392] [General]
Hello! 

I have a question about sequential execution for multiple flows, for example:

I have A flow and B flow. When A flow has been completed without error, i want the Postman to automatically start the B flow. Is this currently possible? Or will it be implemented in future?

**Comment by Ariemeth** [comment]
There is not a way to do that right now.  It has been brought up about eventually allowing flows to call other flows.
